### PR TITLE
fix inspector not showing up in certain cases

### LINKED ISSUE #2496: Inspector not loading: issues with chrome api
there are issues with chrome api. I'm not sure why it errors out for some, and for others is just hidden
Separate issue out of #2491 
**extensions using event pages or service workers...**
onclick event is the issue 

Use generic on click event?
https://github.com/GoogleChrome/chrome-extensions-samples/blob/9aaa5ca35817e6dfa151566f4441a2fe3bf057ff/api-samples/contextMenus/basic/sample.js#L16

**ember tombster not hide/shown:'**
chrome.action.hide is not a function. need to use enable/disable

### PR DESCRIPTION
**Pull request by patricklx**
fix issues with chrome api.
for some the inspector does not show up, for others it does, not sure why.

fix onclick event & hide/show

action icon is always shown
there is no option to hide/show anymore
instead i switch between grey/orange icon to indicate detection.

enable/disable actually only disables click actions

fixes #2496

### CODE REVIEW

File: skeletons/web-extension/manifest.json [hunk 1]
```diff
@@ -39,8 +43,8 @@
 
   "action": {
     "default_icon": {
-      "19": "{{PANE_ROOT}}/assets/images/icon19.png",
-      "38": "{{PANE_ROOT}}/assets/images/icon38.png"
+      "19": "{{PANE_ROOT}}/assets/images/icon19_grey.png",
```
Comment (RobbieTheWagner) [inline skeletons/web-extension/manifest.json:46]: Why are the default icons the grey ones?

File: tests/integration/injection-test.js [hunk 2]
```diff
@@ -66,13 +66,23 @@ class ChromeApi {
...
         delete self.registeredContextMenus[name];
       },
 
       create(menu) {
         self.registeredContextMenus[menu.id] = menu;
+        self.registeredContextMenus[menu.id].onclick = () =>
+          contextMenuListeners.forEach((fn) => fn({ menuItemId: menu.id }));
+      },
+
+      onClicked: {
```
Comment (RobbieTheWagner) [inline tests/integration/injection-test.js:82]: I assume you tested this to ensure it still found the component in the tree and selected it?

File: skeletons/web-extension/manifest.json (hunk 1 above)
Comment (patricklx) [reply skeletons/web-extension/manifest.json:46]: by default it should show `not detected` icon. as the detection state only shows when an ember app is detected

File: tests/integration/injection-test.js (hunk 2 above)
Comment (patricklx) [reply tests/integration/injection-test.js:82]: i tested it now :)
works

**Review by RobbieTheWagner — APPROVED** [review]

**Pull request merged**